BasicSample/Assets/Editor/MixedRealitySampleQuickSetupWindow.cs

```diff
@@ -0,0 +1,237 @@
+                case MixedRealityProjectConfiguration.RunNativelyOnHL2:
+                    targetGroup = BuildTargetGroup.WSA;
+                    EditorUserBuildSettings.SwitchActiveBuildTargetAsync(BuildTargetGroup.WSA, BuildTarget.WSAPlayer);
+                    EditorUserBuildSettings.wsaBuildAndRunDeployTarget = WSABuildAndRunDeployTarget.DevicePortal;
+                    EditorUserBuildSettings.wsaArchitecture = "ARM64";
+                    break;
+                case MixedRealityProjectConfiguration.RunNativelyOnPCVR:
+                    targetGroup = BuildTargetGroup.Standalone;
+                    EditorUserBuildSettings.SwitchActiveBuildTargetAsync(BuildTargetGroup.Standalone, BuildTarget.StandaloneWindows64);
+                    break;
+                case MixedRealityProjectConfiguration.RunRemotelyOnUWP:
+                    remoting = true;
+                    targetGroup = BuildTargetGroup.WSA;
+                    EditorUserBuildSettings.SwitchActiveBuildTargetAsync(BuildTargetGroup.WSA, BuildTarget.WSAPlayer);
+                    EditorUserBuildSettings.wsaBuildAndRunDeployTarget = WSABuildAndRunDeployTarget.LocalMachine;
+                    EditorUserBuildSettings.wsaArchitecture = "Intel64";
+                    // Player Capabilities
+                    UnityEditor.PlayerSettings.WSA.SetCapability(UnityEditor.PlayerSettings.WSACapability.InternetClient, true);
+                    UnityEditor.PlayerSettings.WSA.SetCapability(UnityEditor.PlayerSettings.WSACapability.InternetClientServer, true);
+                    UnityEditor.PlayerSettings.WSA.SetCapability(UnityEditor.PlayerSettings.WSACapability.PrivateNetworkClientServer, true);
+                    break;
+                case MixedRealityProjectConfiguration.RunRemotelyOnWin32:
+                    remoting = true;
+                    targetGroup = BuildTargetGroup.Standalone;
+                    EditorUserBuildSettings.SwitchActiveBuildTargetAsync(BuildTargetGroup.Standalone, BuildTarget.StandaloneWindows64);
+                    break;
+                default:
+                    return;
+            }
+
+            const string AppRemotingPlugin = "Microsoft.MixedReality.OpenXR.Remoting.AppRemotingPlugin";
+            Type appRemotingFeature = typeof(AppRemoting).Assembly.GetType(AppRemotingPlugin);
+            if (appRemotingFeature == null)
+            {
+                Debug.LogError($"Could not find {AppRemotingPlugin}. Has this class been removed or renamed?");
+                return;
+            }
+
+            FeatureHelpers.RefreshFeatures(targetGroup);
+            OpenXRFeature feature = OpenXRSettings.ActiveBuildTargetInstance.GetFeature(appRemotingFeature);
+            if (feature == null)
+            {
+                Debug.LogError($"Could not load {AppRemotingPlugin} as an OpenXR feature. Has this class been removed or renamed?");
+                return;
+            }
+            feature.enabled = remoting? true:false;
```

Comment:
Looks like you can simplify to just passing `remoting` through, since the `true` and `false` cases align.

```suggestion
            feature.enabled = remoting;
```

Reply:
addressed